frame/staking-rewards/src/lib.rs

```diff
@@ -129,32 +132,48 @@ pub mod pallet {
 			position_id: T::PositionId,
 		},
 		/// Split stake position into two positions
+
+		// Review [Andy]
+		// Could we not use a bounded vector or even an array here for the two positions?
+		// Maybe just clearer as:
+		// `SplitPosition { position_one: T::PositionId, position_two: T::PositionId }`
+		// would imply a small refactor...
 		SplitPosition {
 			positions: Vec<T::PositionId>,
 		},
 		/// Reward transfer event.
 		RewardTransferred {
+			// Revuew [Andy]
+			// Maybe we are missing some documentation around the parameters of this event
 			from: T::AccountId,
 			pool: T::RewardPoolId,
 			reward_currency: T::AssetId,
 			/// amount of reward currency transferred.
 			reward_increment: T::Balance,
 		},
 		RewardAccumulationError {
+			// Review [Andy]
+			// Maybe we are missing some documentation around the parameters of this event
 			pool_id: T::RewardPoolId,
 			asset_id: T::AssetId,
 		},
 		MaxRewardsAccumulated {
+			// Review [Andy]
+			// Maybe we are missing some documentation around the parameters of this event
 			pool_id: T::RewardPoolId,
 			asset_id: T::AssetId,
 		},
 		RewardPoolUpdated {
+			// Review [Andy]
+			// Maybe we are missing some documentation around the parameters of this event
 			pool_id: T::RewardPoolId,
```

Comment:
I feel like the fields are fairly self explanatory - any documentation would probably just be "id of the pool that was updated"/"reward asset that  was transferred" etc

`asset_id` could probably be renamed to `reward_asset_id,` though